src/json.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
```

Comment:
GH doesn't show this, but this file was moved to src/core/object/types/json.js (which has the 2015 copyright date) and this file is "new"